test/suite/wtdataset.py

```diff
@@ -72,6 +78,9 @@ def store_one_cursor(self, c, i):
     def store_range(self, key, count):
         c = self.open_cursor()
         for i in range(key, key + count):
+            # Flush the data to tiered storage.
+            if self.tier_share_value != 0 and self.tier_share_value == i:
+                self.testcase.session.flush_tier('force=true')
```

Comment:
I think the new API that gets rid of `flush_tier` is in develop (you may have to merge develop).  Now you can/should do:
```
self.testcase.session.checkpoint('flush_tier=(enabled,force=true)')
```

Reply:
Fixed